packages/amplify-codegen-appsync-model-plugin/src/__tests__/visitors/__snapshots__/appsync-dart-visitor.test.ts.snap

```diff
@@ -0,0 +1,2131 @@
+  @override
+  int get hashCode => toString().hashCode;
+  
+  @override
+  String toString() {
+    var buffer = new StringBuffer();
+    buffer.write(\\"Post {\\");
+    buffer.write(\\"id=\\" + id + \\", \\");
+    buffer.write(\\"title=\\" + title + \\", \\");
+    buffer.write(\\"owner=\\" + owner);
+    buffer.write(\\"}\\");
+    return buffer.toString();
+  }
+  
+  Post copyWith({@required String id, @required String title, @required String owner}) {
+    return Post(
+      id: id ?? this.id,
+      title: title ?? this.title,
+      owner: owner ?? this.owner);
+  }
+  
+  Post.fromJson(Map<String, dynamic> json)  
+    : id = json['id'],
+      title = json['title'],
+      owner = json['owner'];
+  
+  Map<String, dynamic> toJson() =>
+    {'id': id, 'title': title, 'owner': owner};
+
+  static final QueryField ID = QueryField(fieldName: \\"id\\");
+  static final QueryField TITLE = QueryField(fieldName: \\"title\\");
+  static final QueryField OWNER = QueryField(fieldName: \\"owner\\");
+  static var schema = Model.defineSchema(define: (ModelSchemaDefinition modelSchemaDefinition) {
+    modelSchemaDefinition.name = \\"Post\\";
+    modelSchemaDefinition.pluralName = \\"Posts\\";
+    
+    modelSchemaDefinition.authRules = [
+      AuthRule(
+        authStrategy: AuthStrategy.GROUPS,
+        groupClaim: \\"cognito:groups\\",
+        groups: [ \\"admin\\" ],
+        operations: [
+          ModelOperation.CREATE,
+          ModelOperation.UPDATE,
+          ModelOperation.DELETE,
+          ModelOperation.READ
+        ]),
+      AuthRule(
+        authStrategy: AuthStrategy.OWNER,
+        ownerField: \\"owner\\",
+        identityClain: \\"cognito:username\\",
+        operations: [
+          ModelOperation.CREATE,
+          ModelOperation.UPDATE
+        ]),
+      AuthRule(
+        authStrategy: AuthStrategy.PUBLIC,
+        operations: [
+          ModelOperation.CREATE,
+          ModelOperation.UPDATE,
+          ModelOperation.DELETE,
+          ModelOperation.READ
+        ])
+    ];
+    
+    modelSchemaDefinition.addField(ModelFieldDefinition.id());
+    
+    modelSchemaDefinition.addField(ModelFieldDeinition.field(
+      key: Post.TITLE,
+      isRequired: true,
+      ofType: ModelFieldType(ModelFieldTypeEnum.string)
+    ));
+    
+    modelSchemaDefinition.addField(ModelFieldDeinition.field(
+      key: Post.OWNER,
+      isRequired: true,
+      ofType: ModelFieldType(ModelFieldTypeEnum.string)
+    ));
+  });
+}
+
+class PostType extends ModelType<Post> {
+  const PostType();
+  
+  @override
+  Post fromJson(Map<String, dynamic> jsonData) {
+    return Post.fromJson(jsonData);
+  }
+}"
+`;
+
+exports[`AppSync Dart Visitor Model with Connection Directive should generate classes for models with one to many connection 1`] = `
+"import 'package:flutter/foundation.dart';
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:collection/collection.dart';
```

Comment:
Is it possible to sort all the imports before rendering?

Reply:
The first two are base imports for all generated files. The collection is only added when there is list type variable. This part is hard code.

Reply:
We can maintain a set of packages to be imported (including base and additional imports) and sort them before actually rendering them right?

Reply:
Add sort in the new commit